src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -738,7 +738,8 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
                 if (input.is_type<reshape>() || input.is_type<concatenation>())
                     return;
                 auto additional_params_input = activation_node.get_primitive()->additional_params_input;
-                if (activation_func == cldnn::activation_func::relu_negative_slope && additional_params_input.is_valid() &&
+                if (activation_func == cldnn::activation_func::relu_negative_slope &&
+                    (additional_params_input.is_valid() || activation_node.get_primitive()->additional_params.a != 0.0f) &&
```

Comment:
I'm a bit confused here, you are reducing the condition to apply prelu fusion.
Could you also explain the target primitive that does not support prelu fusion? Does it work fine when slope value is not scalar?